### ISSUE TITLE
Staging Review finding: Confusing text alternatives for radiology images

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#99405]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

This ticket was part of a merged Staging Review with MHV Medical Records [Download Records](https://github.com/department-of-veterans-affairs/va.gov-team/issues/92340), [Flow/Images](https://github.com/department-of-veterans-affairs/va.gov-team/issues/95081) and [Settings](https://github.com/department-of-veterans-affairs/va.gov-team/issues/95082)

## Product Information
**Team:** vfs-mhv-medical-records
**Product:** MHV Medical Records
**Feature:** Phase 1 Radiology Flow/Images, Download Records, and Settings

## Findings details
**VA.gov Experience Standard - issue:** [User encounters design components or patterns that are inconsistent or confusing.](https://design.va.gov/about/experience-standards/#user-encounters-consistent-clear-design-components)
**VA.gov Experience Standard - category:** [Consistency](https://design.va.gov/about/experience-standards/#category-consistency)
**Launch-blocking:** Yes
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

When viewing radiology images (eg. `/medical-records/labs-and-tests/[string]/images`), each image is wrapped in a `div` with an `aria-label` of an index number, and then each image has `alt` text with the same index, repeated twice --- but, like, in a more confuisng way.

For example, Image 2 is presented as:
```
<div class="..." aria-label="2">
   <img src="..." alt="2 - 02/01">
</div>
```

If it's not possible to construct a useful text alternative from available metadata, giving alt text as a numerical index is fine. But the alt text still has to make sense! Depending on whether your screen reader announces `aria-label` on a `div` (not all do, because divs aren't interactive), you'll hear that image as either `2, 2, 02 slash 01` or `2, 02 slash 01`. I can't imagine most Veterans will know what to do with that.

## Recommended action

Remove the `aria-label` from the `div` entirely, since it doesn't seem to provide any useful information and screen reader support for `aria-label` on a `div` is mixed.

For the image `alt`, a numerical index is fine but it has to have a comprehensible grammar. In #98846 @BobbyBaileyRB recommends something like "Image [n], details not provided" which I think would be fine here.

## References

- **Accessibility Defect Severity:** [1: Critical. Must be fixed before launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-1)
- **WCAG Success Criteria:** N/A
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by bryan-riley-va** [comment]
Issue resolved as part of MHV-65022 and validated
